src/Tasks/Resources/Strings.resx

```diff
@@ -911,7 +911,6 @@
   </data>
   <data name="GenerateResource.CorruptOutput">
     <value>MSB3555: Output file "{0}" is possibly corrupt.</value>
```

Comment:
```suggestion
     <value>Output file "{0}" is possibly corrupt and will be deleted and recreated.</value>
```

Reply:
Making this change will then require running a build to propagate it to the localized `.xlf`s.